### ISSUE TITLE
Curl install throws sytax error at line 1

### ISSUE THREAD

**Issue by xXACCEXx** [os-js/OS.js#575]
## Expected behaviour
install os.js

## Actual behaviour
fails on line 1 of curl request

```
xxxx@xxxx:~/ # curl -sS https://os.js.org/installer | bash
bash: line 1: syntax error near unexpected token `newline'
bash: line 1: `<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">'
```

## Steps to reproduce the error
run curl script, as discribed in installation docs.
`curl -sS https://os.js.org/installer | sh`

## (Optinal) Browser and OS information
Using ubnutu 16.10, bash:

**Comment by xXACCEXx** [comment]
Why not install via NPM? then use npm install -g?

**Comment by taigrr** [comment]
You're not using the correct URL. Try it without the pipe, and you'll see:

`
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://www.os-js.org/installer">here</a>.</p>
</body></html>
`

So, running the command with the correct URL you'll get:

`curl -sS https://www.os-js.org/installer`

`#!/bin/bash
#
# OS.js automated installer
#
# Simply run this command and you're ready to go
#   curl -sS http://os-js.github.io/OS.js/installer | sh
#


REPO="https://github.com/os-js/OS.js.git"
DEST="OS.js"

if [ -d "$DEST" ]; then
  echo "Destination already exists"
  exit 1
fi

if [ -e /etc/debian_version ]; then
  if ! which npm | grep -s -q "/npm"
  then
    sudo apt-get install -y npm
  fi
  if ! which git | grep -s -q "/git"
  then
    sudo apt-get install -y git
  fi
  if ! which node | grep -s -q "/node"
  then
    sudo apt-get install -y nodejs-legacy
  fi
else
  if ! which npm | grep -s -q "/npm"
  then
    echo "please install npm"
    exit 1
  fi
  if ! which git | grep -s -q "/git"
  then
    echo "please install git"
    exit 1
  fi
  if ! which node | grep -s -q "/node"
  then
    echo "please install nodejs-legacy"
    exit 1
  fi
fi

echo "Downloading OS.js"
git clone --recursive $REPO $DEST

echo "Building"
cd $DEST
npm install --production
node osjs build

echo "INSTALLATION COMPLETE :-)"
echo "Run 'node osjs run' to start the server"
echo "https://os.js.org/manual/"`

Pipe that through bash or sh and it should work properly:

`curl -sS https://www.os-js.org/installer | sh`

**Comment by andersevenrud** [comment]
This was fixed in #576.

I was using the wrong URL in the manual
